R/Connection.R

```diff
@@ -712,6 +712,56 @@ setMethod(
     invisible(TRUE)
   })
 
+#' List Locations of ODBC Configuration Files
```

Comment:
Should be sentence case

Reply:
[680b502](https://github.com/r-dbi/odbc/pull/680/commits/680b50228c201b284d3e93be45b0b2cdcd09cfbb)!